docs/index.rst

```diff
@@ -15,7 +15,7 @@ Contents
     quickstart
     overview
     node
-    v4_migration
```

Comment:
hrm, since v4 is still supported should we keep the `v4_migration` docs?

Reply:
ohh right. Ya, good idea